rts/lra-examples/cdi-participant/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# Quarkus Configuration file
+# key = value
+
+#quarkus.arc.exclude-types=io.narayana.lra.client.internal.proxy.nonjaxrs.LRAParticipantRegistry,io.narayana.lra.filter.ServerLRAFilter,io.narayana.lra.client.internal.proxy.nonjaxrs.LRAParticipantResource
+#quarkus.arc.exclude-types=io.narayana.lra.client.internal.proxy.nonjaxrs.LRAParticipantRegistry,io.narayana.lra.filter.ServerLRAFilter,io.narayana.lra.client.internal.proxy.nonjaxrs.LRAParticipantResource
```

Comment:
Should not be these two lines deleted when they're commented out?

Reply:
+1